samples/oss_common/mariadb.yaml

```diff
@@ -1,5 +1,5 @@
 name: MariaDB Server
 start_cmd: /QOpenSys/pkgs/bin/mysqld_safe --datadir=/QOpenSys/var/lib/mariadb/data
-
+only_if_executable: /QOpenSys/pkgs/bin/mysqld_safe
 check_alive: port
 check_alive_criteria: 3306
```

Comment:
check_alive_criteria is deprecated and should be avoided, right? I fixed this in PR #141.